pytest.ini

```diff
@@ -1,4 +1,5 @@
 [pytest]
 markers =
     unit_tests: Unit tests (deselect with '-m "not unit_tests"')
-    integration_tests: Integration tests (deselect with '-m "not integration_tests"')
\ No newline at end of file
+    integration_tests: Integration tests (deselect with '-m "not integration_tests"')
+asyncio_mode=auto
```

Comment:
pytest warned me about the default being "legacy" asyncio mode.  Recommendation is to at least set to `auto` (and underneath it resolves to legacy).